client/payment-details/dispute-details/dispute-footer.tsx

```diff
@@ -0,0 +1,187 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import moment from 'moment';
+import classNames from 'classnames';
+import { dateI18n } from '@wordpress/date';
+import { getHistory } from '@woocommerce/navigation';
+import { __, sprintf } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { Button, CardFooter, Flex, FlexItem } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import wcpayTracks from 'tracks';
+import type { Dispute } from 'wcpay/types/disputes';
+import { getAdminUrl } from 'wcpay/utils';
+import { formatExplicitCurrency } from 'wcpay/utils/currency';
+import { getDisputeFee, isInquiry } from 'wcpay/disputes/utils';
+import './style.scss';
+
+const formatUnixTimestamp = ( date: string, format: string ) =>
+	dateI18n( format, moment.unix( parseInt( date, 10 ) ).toISOString() );
+
+interface DisputeFooterProps {
+	dispute: Dispute;
+}
+
+const DisputeFooter: React.FC< DisputeFooterProps > = ( { dispute } ) => {
+	const isSubmitted = !! dispute.metadata.__evidence_submitted_at;
+	const isAccepted = dispute.metadata.__closed_by_merchant === '1';
+	const closedDateFormatted = dispute.metadata.__dispute_closed_at
+		? formatUnixTimestamp( dispute.metadata.__dispute_closed_at, 'M j, Y' )
+		: '-';
+
+	const disputeFee = getDisputeFee( dispute );
+	const disputeFeeFormatted = disputeFee
+		? formatExplicitCurrency( disputeFee.fee, disputeFee.currency )
+		: '-';
+	const disputeDocsLinkElement = (
+		// eslint-disable-next-line jsx-a11y/anchor-has-content
+		<a
+			target="_blank"
+			rel="noopener noreferrer"
+			href="https://woocommerce.com/document/woopayments/fraud-and-disputes/"
+		/>
+	);
+
+	let message = null;
+	let buttonLabel = __( 'View dispute details', 'woocommerce-payments' );
+
+	switch ( dispute.status ) {
+		case 'won':
+			message = createInterpolateElement(
+				sprintf(
+					/* Translators: %s - formatted date, <a> - link to documentation page */
+					__(
+						'Good news! You won this dispute on %s. The disputed amount and the dispute fee have been credited back to your account. <a>Learn more about preventing disputes</a>.',
+						'woocommerce-payments'
+					),
+					closedDateFormatted
+				),
+				{ a: disputeDocsLinkElement }
+			);
+			break;
+		case 'under_review':
+			const submissionDateFormatted = dispute.metadata
```

Comment:
These cases feel like sub-components. Maybe you considered that. (Just musing)

Reply:
Now sub-components in d532d23 😉